e2e/index.e2e.js

```diff
@@ -5,7 +5,7 @@ describe('hello, protractor', function () {
     browser.get('/');
     it('should have a title', function () {
       expect(browser.getTitle()).toBe('Material2');
-      screenshot('initial state');
+      screenshot('demo page: index');
```

Comment:
Then do that in this PR if you're going to change the name :)


On Mar 19, 2016, 21:12 -0700, Robert Messerle<notifications@github.com>, wrote:
>  
> Ine2e/index.e2e.js(https://github.com/angular/material2/pull/194#discussion_r56759038):
>  
> >@@ -5,7 +5,7 @@ describe('hello, protractor', function () {>browser.get('/');>it('should have a title', function () {>expect(browser.getTitle()).toBe('Material2');>-   screenshot('initial state');>+   screenshot('demo page: index');
> Ah, good point. I plan to add filtering to that so it will strip special characters. Ideally, I think it would be nice to use slashes... like "component_name/screenshot_name".
> …(#)
> On Sat, Mar 19, 2016 at 9:09 PM, Hans<notifications@github.com>wrote: In e2e/index.e2e.js<#194 (comment)(https://github.com/angular/material2/pull/194#discussion_r56759019)>:>@@ -5,7 +5,7 @@ describe('hello, protractor', function () {>browser.get('/');>it('should have a title', function () {>expect(browser.getTitle()).toBe('Material2');>- screenshot('initial state');>+ screenshot('demo page: index'); Is that still going in the file name? If so Windows is allergic to :s — You are receiving this because you authored the thread. Reply to this email directly or view it on GitHub<https://github.com/angular/material2/pull/194/files/d34400bbbf1171b866c75802d269e0fd830c6c93#r56759019>
>  
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly orview it on GitHub(https://github.com/angular/material2/pull/194/files/d34400bbbf1171b866c75802d269e0fd830c6c93#r56759038)
>  
>  
>